docs/installation/nuxt.md

```diff
@@ -80,15 +80,23 @@ Now, let’s replace the content of `pages/index.vue` with the following example
 
 ```html
 <template>
-  <div id="app">
+  <div>
```

Comment:
Do we need the `<div>` then?

Reply:
We do need it, because of the `<client-only>` tag afterwards. If we do not keep the div the page will have no actual wrapper when rendered in the server side.